files/image_config/memory_statistics/memory_statistics_handler.py

```diff
@@ -0,0 +1,1877 @@
+#!/usr/bin/env python3
+import psutil
```

Comment:
Can you sort import as nit purpose?

Reply:
Thank you for the review. I have organized the imports following the standard Python import grouping practice: standard library imports, third-party imports, and local application imports, sorted alphabetically within each group for improved readability and maintainability.